src/components/Input.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useEffect, useState } from "react"
+import { makeStyles } from "@material-ui/core/styles"
+import TextField from "@material-ui/core/TextField"
+import PrimaryButton from "../UI/PrimaryButton"
+import { RegexContext } from "../store/regex-context"
+import ConfettiForWinner from "../UI/Confetti"
+import { setTimeEvent } from "../functions/setTimeoutFunc"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "& > *": {
```

Comment:
try to avoid things like this in child components. you should add global style only in the most "parent" component